src/resolvers/guidance.ts

```diff
@@ -281,15 +213,18 @@ export const resolvers: Resolvers = {
     guidanceGroup: async (parent: Guidance, _, context: MyContext): Promise<GuidanceGroup> => {
       return await GuidanceGroup.findById('Chained Guidance.guidanceGroup', context, parent.guidanceGroupId);
     },
-    // Chained resolver to fetch the Tags for this Guidance
```

Comment:
Do we still want a chained resolver to the `Tag` using the new `tagId` foreign key?